lib/config/migrations/custom/host-rules-migration.ts

```diff
@@ -25,17 +33,25 @@ export class HostRulesMigration extends AbstractMigration {
           key === 'domainName'
         ) {
           if (is.string(value)) {
-            newRule.matchHost ??= value;
+            newRule.matchHost ??= massageUrl(value);
           }
           continue;
         }
 
         newRule[key] = value;
       }
 
-      return newRule;
-    });
+      newHostRules.push(newRule);
+    }
 
     this.rewrite(newHostRules);
   }
 }
+
+function massageUrl(url: string): string {
```

Comment:
shouldn't this be a private static class method? 🤔

Reply:
> Consider converting private methods to non-exported functions within the same file but outside of any class, and moving private properties into a separate, non-exported class.


https://google.github.io/styleguide/tsguide.html#visibility

but im not zealot about it :)

Reply:
ok. 😅